### PR TITLE
Added convertkit subscription conponent

### PR DESCRIPTION
**Pull request by okanji**

### CODE REVIEW

**Review by calebissharp — CHANGES_REQUESTED** [review]
It looks like the form doesn't quite fit on mobile displays:

image

File: src/components/NewsLetter.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+        <Input
+          type="email"
+          aria-label="Your email"
+          //The name attribute should be the same as on you selected form.
+          name="email_address"
+          placeholder="Your email address"
+          onChange={handleInputChange}
+          value={email}
+          placeholder="Enter Name"
+          error={status === 'ERROR' || validateInput(email)}
```
[the remark this answers is not in the document]
Comment (okanji) [reply src/components/NewsLetter.js]: Agreed

**Comment by okanji** [comment]
@calebissharp I have made the changes per your review, thanks for pointing those things out. I have tested the app using a test deployment with Vercel. For some reason it only works when the convertkit ID is hardcoded, when I set it as an env variable it does not seem to work even though I have set the env variable in the vercel deployment. I am not sure how to test this further, maybe its just an issue with vercel but will work in our production environment?

**Comment by calebissharp** [comment]
@okanji I think the env var has to be set at build time, if that has anything to do with it.

**Comment by okanji** [comment]
@calebissharp ok I see, so this should work in production?

**Comment by calebissharp** [comment]
@okanji IIRC we use semaphore to run the build and then upload it to S3. So you will have to add the environment variables inside semaphore.

**Pull request merged**